serverless/sql-databases/quickstart.mdx

```diff
@@ -0,0 +1,95 @@
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+  - You have installed [PostgreSQL](https://www.postgresql.org/download/).
+</Message>
+
+Scaleway Serverless SQL Databases are fully managed databases that automatically scale both in storage and compute according to your needs.
+
+This page explains how to create, access and delete a Serverless SQL Database using the Scaleway Console.
+
+## How to create a database
+
+1. Click **SQL Databases** in the **Serverless** section of the side menu. The Serverless SQL Databases page displays.
+
+2. Click **+ Create database**. The database creation wizard displays.
+
+3. Complete the following steps in the wizard:
+
+    - Choose a **region**, which is the geographical location in which your database will be created.
+
+    - Select a **default engine version** from the drop-down menu.
+
+    - Configure your **database autoscaling**, or keep the default values. Your database will scale within the defined range to match the incoming load.
+
+    <Message type="note">
+        Define a minimum of 1 vCPU or more to avoid cold starts that can take a few seconds. 
+    </Message>
+
+    - Enter a **name** for your database, or keep the randomly-generated name. The name must only contain alphanumeric characters and dashes.
+
+    - Verify the **estimated cost**.
+
+4. Click **Create database** to finish. The setting up of your database will take a few minutes. Check the **Overview** tab to see its status.
+
+## How to connect to a database
+
+1. From the **Overview** tab of your database, click the **Set up credentials** button. The wizard displays.
+
+2. Select yourself as the [principal](/identity-and-access-management/iam/concepts/#principal) to connect to the database.
+
+<Message type="note">
+    Refer to the [IAM documentation](/identity-and-access-management/iam/reference-content/overview/) for more information on identities and access rights.
+</Message>
+
+3. Set up the **Connection string**:
+
+    - If you already have a secret key, paste it in the connection string in place of the``[INSERT_SECRET_KEY_HERE]`` placeholder.
+
+    - If you do not have a secret key, click **Generate new secret key**.
+
+4. Copy your connection string and store it securely.
+
+5. In a terminal, run the following command:
+
+    ```bash
+    psql "[YOUR_CONNECTION_STRING]"
+    ```
+
+An output similar to the following response displays:
+
+    ```bash
+    psql (15.3, server 14.8 (Debian 14.8-1pgsg120+1))
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_128_GCM_SHA256, compression: off)
+    Type "help" for help.
+    ```
+
+## How to delete a database
+
+1. Click **SQL Databases** in the **Serverless** section of the side menu. The Serverless SQL Databases page displays.
+
+2. Click the <Icon name="more" /> icon next to the database you want to delete, and select **Delete** from the dropdown menu.
```

Comment:
```suggestion
2. Click the <Icon name="more" /> icon next to the database you want to delete, and select **Delete** from the dropdown menu. A pop-up displays.
```

I'd put what happens right after describing the action but it might be a preference, feel free to choose what sounds best to you :)